tst/regression/test_suites/amr_performance/parthinput.amr_performance

```diff
@@ -0,0 +1,75 @@
+#========================================================================================
+# Parthenon performance portable AMR framework
+# Copyright(C) 2021 The Parthenon collaboration
+# Licensed under the 3-clause BSD License, see LICENSE file for detail
+# ========================================================================================
+#  (C) (or copyright) 2021. Triad National Security, LLC. All rights reserved.
+#
+#  This program was produced under U.S. Government contract 89233218CNA000001 for Los
+#  Alamos National Laboratory (LANL), which is operated by Triad National Security, LLC
+#  for the U.S. Department of Energy/National Nuclear Security Administration. All rights
+#  in the program are reserved by Triad National Security, LLC, and the U.S. Department
+#  of Energy/National Nuclear Security Administration. The Government is granted for
+#  itself and others acting on its behalf a nonexclusive, paid-up, irrevocable worldwide
+#  license in this material to reproduce, prepare derivative works, distribute copies to
+#  the public, perform publicly and display publicly, and to permit others to do so.
+# ========================================================================================
+
+<parthenon/job>
+problem_id = advection
+
+<parthenon/mesh>
+refinement = adaptive
+numlevel = 4
+
+nx1 = 128
+x1min = -1.50
+x1max = 1.50
+ix1_bc = periodic
+ox1_bc = periodic
+
+nx2 = 128
+x2min = -1.50
+x2max = 1.50
+ix2_bc = periodic
+ox2_bc = periodic
+
+nx3 = 128
+x3min = -1.50
+x3max = 1.50
+ix3_bc = periodic
+ox3_bc = periodic
+
+<parthenon/meshblock>
+nx1 = 32
+nx2 = 32
+nx3 = 32
+
+<parthenon/time>
+tlim = 1.0
+integrator = rk1
+nlim = 10
+ncycle_out_mesh=-10
+
+<Advection>
+cfl = 0.30
+vx = 1.0
+vy = 1.0
+vz = 1.0
+profile = smooth_gaussian
```

Comment:
Not that it matters much, but is there a reason you chose the gaussian here and not the hard sphere?